server/form-pages/apply/reasons-for-placement/basic-information/endDates.ts

```diff
@@ -54,6 +59,14 @@ export default class EndDates implements TasklistPage {
   errors() {
     const errors: TaskListErrors<this> = {}
 
+    ;['sedDate', 'ledDate', 'pssDate'].forEach(date => {
+      if (!dateIsBlank(this.body, date)) {
+        if (!dateAndTimeInputsAreValidDates(this.body, date)) {
+          errors[date] = 'Date must be a valid date'
```

Comment:
It'd be good if we could return unique errors for each field, maybe with a lookup or something?